apps/files/js/files.js

```diff
@@ -25,7 +25,9 @@
 				state.call.abort();
 			}
 			state.dir = currentDir;
-			state.call = $.getJSON(OC.filePath('files','ajax','getstoragestats') + '?dir=' + encodeURIComponent(currentDir),function(response) {
+			state.call = $.getJSON(OC.generateUrl('apps/files/ajax/getstoragestats', {
+				dir: currentDir,
+			}), function(response) {
```

Comment:
generateUrl second param is a placeholder replacement, not a list of GET params.

```suggestion
			state.call = $.getJSON(OC.generateUrl('apps/files/ajax/getstoragestats?dir={dir}', {
				dir: currentDir,
			}), function(response) {
```